src/components/Search.vue

```diff
@@ -21,6 +22,8 @@ function handleInput(e) {
   emit('update:modelValue', input);
 }
 
+const handleInputDebounce = debounce(handleInput, 300);
```

Comment:
Can you try lower the debounce time like this?
```suggestion
const handleInputDebounce = debounce(handleInput, 50);
```